_src-lp/blocks/banner/banner.css

```diff
@@ -385,4 +396,21 @@
     margin-bottom: 0 !important;
     padding-bottom: 0 !important;
   }
+}
+@media screen and (width <= 991px) {
+  .section.banner-container.hide-tablet {
+    background-image: unset !important;
+    background-color: #000;
+  }
+}
+@media screen and (width >= 1400px) {
+  .section.banner-container .custom-col-xl-4 {
+    flex: 0 0 auto;
+    width: 27.33%;
+  }
+}
+@media screen and (width >= 1800px) {
```

Comment:
1800 ? why ? which viewport case it this ? I don't think we should have this at all